lms/static/js/spec/student_account/account_settings_view_spec.js

```diff
@@ -49,6 +49,32 @@ define(['backbone',
                                 })
                             }
                         ]
+                    },
+                    {
+                        title: 'Social Links',
```

Comment:
@HarryRein  Should this title be "Social Media Links" rather than "Social LInks"?